install/playbooks/roles/system-post-install/tasks/set-limits.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: Set limits per user
+  notify:
+    - Restart Postfix
+    - Restart Dovecot
+  template:
+    src: limits.conf
+    dest: /etc/security/limits.conf
```

Comment:
There seems to be a `/etc/security/limits.d/` directory. Files named `*.conf` are taken into account.

Maybe it could go in a `/etc/security/limits.d/homebox.conf` for example?

Reply:
Done, thanks.